STMicroelectronics/STM32L4_L4+/app/az_ulib_dm/src/az_ulib_dm.c

```diff
@@ -69,7 +70,29 @@ AZ_NODISCARD az_result az_ulib_dm_init(az_ulib_dm* dm_handle)
     az_span_fill(package->name, '\0');
   }
 
-  return _az_ulib_dm_interface_publish();
+  az_result result;
```

Comment:
>az_result result; [](http://example.com/codeflow?start=2&length=17)

we usually declare the result just after the precondition checks.

Reply:
did not mean to commit these couple lines, meant for my testing only. Will change everything back to the way it was!